test/quickcheck/main.cc

```diff
@@ -0,0 +1,23 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include <gtest/gtest.h>
+
+#include <cassert>
+
+#include "env.h"
+#include "monitor.h"
+
+int main(int argc, char **argv) {
+  Watchdog *watchdog = Watchdog::Create("/tmp/stacktrace.cvmfs_unittests");
```

Comment:
We might want to store the stack trace under `/tmp/stacktrace.cvmfs_qc` or similar